components/containers/DynamicForm/Settings.tsx

```diff
@@ -51,20 +53,32 @@ export const FormSettings = (props: FormSettingsProps): React.ReactElement => {
   return (
     <>
       <h1 className="gc-h1">{t("settings.title")}</h1>
-      <div>{newText}</div>
       <div data-testid="formID" className="mb-4">
         Form ID: {form.formID}
       </div>
-      <h2>Edit Form Config File:</h2>
-      <JSONUpload form={form}></JSONUpload>
-      <br />
-      <div>
-        <DeleteButton
-          action={handleDelete}
-          data={form.formID}
-          redirect={`/admin/view-templates`}
-        ></DeleteButton>
-      </div>
+      <Tabs>
+        <TabList>
+          <Tab>{t("settings.tabLabels.jsonUpload")}</Tab>
+          <Tab>{t("settings.tabLabels.bearerToken")}</Tab>
+        </TabList>
+
+        <TabPanel>
+          <div>{newText}</div>
+          <h2>{t("settings.edit")}</h2>
+          <JSONUpload form={form}></JSONUpload>
```

Comment:
We should align with the new props requirement and only be passing in the `formID` instead of the complete `form` object.

Reply:
Great idea; I think that's outside the scope of this PR though, as it's changing the `JSONUpload` component, which is otherwise unchanged.

Reply:
Agreed.  This is my bad, when I was going through the components tree I mistook JSONUpload for the new component that is introduced in this PR.